frame/multisig/src/lib.rs

```diff
@@ -434,7 +433,6 @@ pub mod pallet {
 
 			T::WeightInfo::approve_as_multi_create(s)
 				.max(T::WeightInfo::approve_as_multi_approve(s))
-				.max(T::WeightInfo::approve_as_multi_complete(s))
```

Comment:
I am not sure if this is correct. @ruseinov do you remember why the `approve_as_multi_complete` benchmark was removed?  
Why do we not need it anymore?

Reply:
Okay seems fine since the bench does not make sense anymore.